Traceback (most recent call last):
  File "thickness_custom.py", line 117, in <module>
    c.execute(query);
sqlite3.OperationalError: database is locked
